CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 16.21.0
 
 - Remove links to Whitehall publications (PR #823)
```

Comment:
I think this went into 16.20.1 :-|.

Reply:
Right. Shuffled entries to match reality.